.gitignore

```diff
@@ -2,4 +2,6 @@
 !CHANGELOG.MD
 !.gitignore
 !screenshots.png
-!README.md
\ No newline at end of file
+!README.md
+!.github/
+!.github/issue_template.md
```

Comment:
To add an exception for a file inside a sub-folder, we need 2 rules. One for the folder itself and one for the file we want the exception for. Tested that only this file in this sub-folder is being included